beacon_node/network/src/sync/backfill_sync/mod.rs

```diff
@@ -557,7 +558,7 @@ impl<T: BeaconChainTypes> BackFillSync<T> {
             }
         };
 
-        let Some(peer) = batch.processing_peer() else {
+        let Some(batch_peers) = batch.processing_peers() else {
```

Comment:
You can not log the batch_peers in the log below as they are already logged when the batch completes downloading. Then you can remove the error condition below. It's odd to fail sync in this case when below in the other call to `processing_peers` it's just a warn